app/src/main/res/drawable/background_square.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="?colorSurface" />
+    <solid android:color="#ff00ff" />
```

Comment:
Is this correct?

Reply:
Yes! As I intended for this to only be used when we have to use `colorStateList`, so that it is apparent when it is not working

Reply:
Big brain-move right there, I like it!